src/libstd/sync/mutex.rs

```diff
@@ -107,6 +107,61 @@ use crate::sys_common::poison::{self, LockResult, TryLockError, TryLockResult};
 ///
 /// *guard += 1;
 /// ```
+///
+/// It is sometimes a good idea (or even necessary) to manually drop the mutex
+/// to unlock it as soon as possible. If you need the resource until the end of
```

Comment:
It is not the mutex that is being dropped.

```suggestion
/// guard to unlock it as soon as possible. If you need the resource until the end of
```